assets/src/components/AvatarModal.js

```diff
@@ -84,18 +132,12 @@ function AvatarModal(props) {
             <Divider />
             {
               user.admin
-                ?
-                <>
-                  <Link style={{ textDecoration: 'none' }} href="/admin">
-                    <ListItem button>
-                      <ListItemIcon>
-                        <Lock />
-                      </ListItemIcon>
-                      <ListItemText inset primary='Admin' />
-                    </ListItem>
-                  </Link>
-                  <Divider />
-                </>
+                ? Admin
+                : null
+            }
+            {
+              user.enrolledCourses.length > 1
```

Comment:
I only show the `SwitchCourses` component if the user is enrolled in more than one course. However, if they are on course 1234, and they click to switch to course 1234, the entire app reloads instead of simply closing the dialog box. Again, this can be addressed, just takes a bit more work.

Reply:
Yeah, we might fix it later also what to get feedback from testing this feature. we haven't had a situation where  2 users are in a different courses enabled with Myla tool

Reply:
Sounds good.